README.md

```diff
@@ -177,6 +177,7 @@ _Default:_ `false`
 
 ### Projects
 
+* [React, Universally](https://github.com/ctrlplusb/react-universally)
```

Comment:
It's just probably good idea to add new items to the end of the list (to be fair)